src/components/Header/Header.tsx

```diff
@@ -43,17 +55,29 @@ export const Header = ({ ...rest }): React.ReactElement => {
   const [show, setShow] = useState(false)
   const toggleMenu = () => setShow(!show)
 
+  useEffect(() => {
+    setShow(false)
+  }, [])
+
```

Comment:
This ends up being redundant, since default state on component mounting is set the same anyway